### DISCUSSION TITLE
How to use MindsDB to connect to Dremio?

### DISCUSSION THREAD

**Discussion by YishuiLi** [mindsdb/mindsdb#4645] [Get help] [answered]
I tried connect to Dremio using MindsDB but failed. Does Mindsdb presently permit connections to Dremio? How to use MindsDB to connect to Dremio?

**Comment by ZoranPandovski** [comment, marked as answer]
Hey, atm MindsDB doesn't support Dremio. We just got a PR with the support that needs to be reviewed https://github.com/mindsdb/mindsdb/pull/4637. Once merged, should be ready to use

**Reply by YishuiLi** [reply]
Got it! Thanks!
